front-ps/src/Guide/Api/Http.purs

```diff
@@ -33,9 +32,9 @@ fetchGrandCategories (CategoryName catName) = do
   res <- attempt <<< fetch $ apiEndpoint <> catName <> "/all-categories"
   pure $ either (Left <<< show) decodeJson res
 
--- TODO: Use `Uid Category` instead of `String` as second function parameter
+-- TODO: Use `CUid Category` instead of `CUid String` as second function parameter
 -- if we have found a way to bridge `Uid a` properly from `Haskell` to `PS`
-fetchCategory :: forall eff. CategoryName -> (Uid String) -> Aff (fetch :: FETCH | eff) (Either String CCategoryDetail)
-fetchCategory (CategoryName catName) (Uid catId) = do
+fetchCategory :: forall eff. CategoryName -> (CUid String) -> Aff (fetch :: FETCH | eff) (Either String CCategoryDetail)
+fetchCategory (CategoryName catName) (CUid catId) = do
   res <- attempt <<< fetch $ apiEndpoint <> catName <> "/category/" <> catId.uidToText
```

Comment:
It make no sense to have `CUid String` and then using ` uidToText` (`catId.uidToText`) to get this string. It would be nice just to grab `catId` from `CUid catId` to get the string. Maybe `newtype CUid = CUid String` could fix this (see my other comment below).